sormas-ui/src/main/java/de/symeda/sormas/ui/person/PersonController.java

```diff
@@ -80,13 +80,23 @@ public TitleLayout getPersonViewTitleLayout(PersonDto personDto) {
 	}
 
 	public void selectOrCreatePerson(final PersonDto person, String infoText, Consumer<PersonReferenceDto> resultConsumer, boolean saveNewPerson) {
+		selectOrCreatePerson(person, infoText, resultConsumer, saveNewPerson, false, null);
+	}
+
+	public void selectOrCreatePerson(
+		final PersonDto person,
+		String infoText,
+		Consumer<PersonReferenceDto> resultConsumer,
+		boolean saveNewPerson,
+		boolean showWithoutMatches,
```

Comment:
It seems to me that this boolean is not necessary. It can be determined via `StringUtils.isEmpty(infoTextWithoutMatches)`.

Reply:
makes sense